### ISSUE TITLE
onSwipeSuccess can not be used with params, always remembers initial state

### ISSUE THREAD

**Issue by chabanenkopp** [UdaySravanK/RNSwipeButton#48]
**Describe the bug**
onSwipeSuccess doesn't work properly when you want to call it with params. In my case I would like to call an async function and pass it email and password. Unfortunately, in the current implementation of onSwipeSuccess it "remembers" the function with just default values that you pass to it. 

```
  onSwipeSuccess={() => 
                signInWithEmailAndPassword({ email, password })
              }
```

in the example above the email and password params come from React.useState: 

```
 const [email, setEmail] = React.useState('')
 const [password, setPassword] = React.useState('')
```

where setEmail and setPassword are passed as onChangeText to react native TextInput component. 
When I call **signInWithEmailAndPassword** as onPress with a simple button, it works fine. 

For example, if you use a simple console.log like that: 

```
  onSwipeSuccess={() => 
              console.log({ email, password })
              }
```

you will always have the following result: email: "", password: "", no matter how the email and password values are changed via Text Inputs, because onSwipeSuccess accepts the function passsed to it just once and doesn't rerender it...

**Expected behavior**
onSwipeSuccess accepts a function with parameters according to the latest state of these pareameters without memozining the initially passed function (or whatever it does to it now...)

**Comment by mohamedmaged** [comment]
we have the same issue we don't why this is hapenning

**Comment by mohamedmaged** [comment]
@UdaySravanK  do you know why this is happenning ? or any hotfix we can use ?

**Comment by mohamedmaged** [comment]
@chabanenkopp  hello I fixed this by making  a state like this 

`  const [swipeSuccess,setSwipeSuccess] = useState(false)
useEffect(()=>{
    signInWithEmailAndPassword({ email, password })
    setSwipeSuccess(false)
  },[swipeSuccess])

  const actionSwipeSuccess = ()=>{
    setSwipeSuccess(true)
  }
`
and then at the onSwipeSuccess cal the function that changes the swipeSucces value 
 onSwipeSuccess={() =>actionSwipeSuccess()  }

**Comment by UdaySravanK** [comment]
Hi @chabanenkopp , This issue must have been fixed in latest version [v1.3.3](https://github.com/UdaySravanK/RNSwipeButton/releases/tag/v1.3.3)
Please check. Thanks for reporting the issue.

CC: @mohamedmaged
